bots/images/scripts/lib/atomic.install

```diff
@@ -166,23 +166,38 @@ class AtomicCockpitInstaller:
         """ Install the latest cockpit RPMs in our container"""
         rpm_args = []
         for package in self.rpms:
-            if 'cockpit-ws' in package or 'cockpit-dashboard' in package or 'cockpit-bridge' in package:
+            if 'cockpit-ws' in package or 'cockpit-dashboard' in package or 'cockpit-bridge' in package or 'libssh' in package:
                 rpm_args.append("/host" + package)
 
         if rpm_args:
             subprocess.check_call(["docker", "run", "--name", "build-cockpit",
                                    "-d", "--privileged", "-v", "/:/host",
                                    "cockpit/ws", "sleep", "1d"])
-            if rpm_args:
-                rpm_args = ["docker", "exec", "build-cockpit",
-                            "rpm", "--freshen", "--verbose", "--force"] + rpm_args
-                if self.verbose:
-                    print("updating cockpit-ws container")
-                subprocess.check_call(rpm_args)
-
-                # if we update the RPMs, also update the scripts, to keep them in sync
-                subprocess.check_call(["docker", "exec", "build-cockpit", "sh", "-exc",
-                                       "cp /host/var/tmp/containers/ws/atomic-* /container/"])
+            if self.verbose:
+                print("updating cockpit-ws container")
+
+            to_install = []
+            to_freshen = []
```

Comment:
Why do we need to use both `--install` and `--freshen`?  Can't we just use `--install` with all packages?

Reply:
Using `--install (--force)` fails with some of the packages, because the newer cockpit packages could require newer/more dependencies. For the same reason `--upgrade (--force)` fails too; `--freshen` will just do nothing and exit 0 if dependencies are not met.

Reply:
I see, thanks.  Wouldn't it actually be better top fail if we can't install a package? But this PR is not about that.